types/react-loadable/index.d.ts

```diff
@@ -0,0 +1,118 @@
+// Type definitions for react-loadable 3.3
+// Project: https://github.com/thejameskyle/react-loadable#readme
+// Definitions by: Diogo Franco <https://github.com/Kovensky>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as React from 'react';
+
+// tslint:disable-next-line strict-export-declare-modifiers
+type LoadedComponent<Props> = React.ComponentClass<Props> | React.SFC<Props>;
```

Comment:
This is just a shorthand alias for internal use, and not something I want to export; but the strict-export-declare-modifiers was demanding I added an `export` here.

If this is NG, then I'll have to change all instances of `LoadedComponent<Props>` into `(React.ComponentClass<Props> | React.SFC<Props>)` which can lead to some very long lines in the interfaces.